ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -214,7 +214,7 @@
         }
       ],
       "attributes": {
-        "plugins": "eclipse/che-machine-exec-plugin/0.0.1",
+        "plugins": "eclipse/che-machine-exec-plugin/next",
         "editor": "eclipse/che-theia/1.0.0"
```

Comment:
Shouldn't this also be `eclipse/che-theia/next`?

Reply:
Machine exec should be `0.0.1` and `che-theia` - `next`, and now they are such.